server/test/api/grades.js

```diff
@@ -0,0 +1,94 @@
+var chai = require('chai');
+var expect = chai.expect;
+var uuid    = require('node-uuid');
+
+var helpers = require('./helpers');
+helpers.configure(chai);
+
+/**
+* The /grade   API endpoint
```

Comment:
This comment says `/grade`.  It will not block this PR, but please take more care in the future...